Assets/__Scripts/UI/Options/AdbHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using QuestDumper;
+using UnityEngine;
+
+namespace __Scripts.UI.Options
```

Comment:
Classes in `__Scripts` don't have a namespace (mostly a Unity moment). This should be removed for consistencies' sake.